packages/deploy-aws/test/templates/seagull_project.ts

```diff
@@ -44,4 +44,38 @@ export class Test extends BasicTest {
     const synthStack = project.synthesizeStack('helloworld')
     Object.keys(synthStack.template.Resources).length.should.be.above(1)
   }
+
+  @test
+  async 'can create a project with customized stack'() {
```

Comment:
what sould happen if the files does not exist or is corrupted?

Reply:
See the new tests :)